tests/phpt/string_functions/010_xor_strings.php

```diff
@@ -0,0 +1,24 @@
+@ok
+<?php
+
+function is_kphp() {
+#ifndef KPHP
+    return false;
+#endif
+    return true;
+}
+
+function test_xor_strings() {
+  $strings = ["handling and manipulating", "ings, take a look", "ctory Access Protocol", "0000000dwxasd", "1111sadsd11"];
```

Comment:
let's add a single char string to the array, because the current test succeeds even without a bug fix